api/project.py

```
@@ -0,0 +1,112 @@
+    if path.exists():
+        shutil.rmtree(path)
+
+    # Create directory
+    path.mkdir(parents=True, exist_ok=True)
+
+    # Clone repository
+    # Prepare the Git clone command
+    cmd = ["git", "clone", url, path]
+
+    # Run the git clone command and wait for it to finish
+    result = subprocess.run(cmd, check=True, capture_output=True, text=True)
```

Ruff (0.8.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)